source/sass/main.scss

```diff
@@ -327,23 +328,25 @@ button {
     .container {
       margin-top: 0;
       margin-bottom: 0;
-      min-height: 220px;
-
-      @media screen and (min-width: $bp-md) {
-        display: flex;
-        justify-content: space-around;
-        align-items: center;
-      }
     }
 
     .info-box {
-      margin: 1.5rem 0;
-      display: inline-block;
-      width: 100%;
-      max-width: 342px;
-      font-size: 24px;
-      font-weight: 500;
-      line-height: 33px;
+      margin-top: 1.5rem;
+
+      &:not(:first-child) {
```

Comment:
@ScottDowne feel free to submit a patch to fix this. I don't know how to get collapsing margins to work on these elements... :nauseated_face: